src/main/scala/akka/persistence/cassandra/query/javadsl/CassandraReadJournal.scala

```diff
@@ -145,5 +147,45 @@ class CassandraReadJournal(scaladslReadJournal: akka.persistence.cassandra.query
    */
   def currentEventsByTag(tag: String, offset: UUID): Source[UUIDEventEnvelope, Unit] =
     scaladslReadJournal.currentEventsByTag(tag, offset).asJava
+
+  /**
+    * `eventsByPersistenceId` is used to retrieve a stream of events for a particular persistenceId.
+    *
+    * In addition to the `offset` the `EventEnvelope` also provides `persistenceId` and `sequenceNr`
```

Comment:
Currently the offset is zero based and sequenceNr one based. E.g. when offset is 0, sequenceNr is 1. When offset is 1, sequenceNr is 2 etc. Do you think they should be the same? Offset is not used in the eventsByPersistenceId query anyway as far as I am aware.